### PR TITLE
Update for bevy 0.11

### PR DESCRIPTION
**Pull request by ostwilkens**
Tested on windows 10. 
Had to fix update_joystick_by_mouse as it was only triggering if the mouse moved the same frame as the click. (Events might be flushed each frame now, even if not consumed?)
I'm not 100% sure if the ordering of the systems are correct with the new terminology. 
Examples need to be updated as well. 
Feel free to edit, rip or rewrite.

### CODE REVIEW

**Comment by SergioRibera** [comment]
Ooooh yes, I was waiting for them to update the `bevy-editor-egui` to upload the changes, didn't you have any problems when trying it?

**Comment by ostwilkens** [comment]
If you mean `bevy-inspector-egui` it's up-to-date :-)

**Comment by SergioRibera** [comment]
@ostwilkens Oh great, can you fix the checks please run `cargo make` or `cargo fmt --all -- -- --check && cargo clippy -- -D warnings` to fix the errors you have and I'll approve the PR

File: src/input.rs [hunk 1]
```diff
@@ -137,33 +138,33 @@ pub fn update_joystick(
 
 pub fn update_joystick_by_mouse(
     mouse_button_input: Res<Input<MouseButton>>,
-    mut cursor_evr: EventReader<CursorMoved>,
+    mut mousebtn_evr: EventReader<MouseButtonInput>,
```
Comment (SergioRibera) [inline src/input.rs:141]: This does not alter the current operation? is everything correct with the examples?

**Comment by ostwilkens** [comment]
I haven't used this plugin before 0.11, so I'm not qualified to fix/test the examples. I don't know how things are supposed to behave 😅

**Comment by SergioRibera** [comment]
> I haven't used this plugin before 0.11, so I'm not qualified to fix/test the examples. I don't know how things are supposed to behave 😅

Don't worry, I'll try it

**Comment by ostwilkens** [comment]
I started trying to update the simple_pc example, but got stuck. Where is `JoystickController` defined?

**Comment by SergioRibera** [comment]
> I started trying to update the simple_pc example, but got stuck. Where is `JoystickController` defined? 

Noooo, hahahaha, I just noticed, I don't know when that slipped in, well it shouldn't be `JoystickController` but `String`

File: src/input.rs (hunk 1 above)
Comment (SergioRibera) [reply src/input.rs:148]: I just tested it and it works great, thank you very much for the fix

**Comment by SergioRibera** [comment]
@ostwilkens I just tested and it seems that everything is ok, do you want to continue updating the examples or would you prefer me to do the merge and do it myself?

**Comment by ostwilkens** [comment]
There we go! 
Just a note, is there a reason for not using the cargo standard example structure? The current setup means a separate build for each example; my virtual_joystick folder is now 22gb :P

**Comment by SergioRibera** [comment]
Thank you very much for your collaboration, I will take into account what you said here, I hope to see you more often here collaborating and I also hope that at some point the library will be useful to you :D

**Comment by SergioRibera** [comment]
@ostwilkens the new version was released [2.0.1](https://crates.io/crates/virtual_joystick)

**Pull request merged**